backend/src/api/user.rs

```diff
@@ -0,0 +1,50 @@
+use actix_web::error::{ ErrorNotFound, ErrorInternalServerError };
+use actix_web::{ get, post, web, Responder };
+use serde::Deserialize;
+
+use crate::extractors::auth::AuthenticatedUser;
+use crate::model::user::User;
+use crate::db::util::DbPool;
+
+#[get("/user/{uid}")]
+pub async fn get_user(uid: web::Path<String>, pool: web::Data<DbPool>) -> impl Responder {
+    User::find_by_uid(pool.get_ref(), uid.as_ref()).await
```

Comment:
For later: What do you think about making a function for this? Since, we will use this in a lot of places I imagine.

Reply:
i suspect you're right, but i tend to wait until refactoring like that's needed before writing it -- DRY doesn't count if we haven't repeated ourselves yet :P